### PR TITLE
Add license to POD, fix pod and add test

### PR DESCRIPTION
**Pull request by lharey**
Hi

As per our emails re the Pull Request challenge.
This fixes the cpants warning about license and fixes the pod error in pmake.
I've also added a pod test and also fixed an error with an incorrectly named file in the manifest

Regards
Lisa

### CODE REVIEW

File: t/97-pod.t [hunk 1]
```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env perl
+
```
Comment (klp2) [inline t/97-pod.t]: Can you add use strict/use warnings here?

File: t/97-pod.t [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+use Test::More;
+eval "use Test::Pod 1.00";
```
Comment (klp2) [inline t/97-pod.t]: Can you add a ##no critic before the eval?

**Comment by lharey** [comment]
oops sorry I always use strict/warnings don't know why I missed it here.

Fixed now

**Comment by klp2** [comment]
Thanks for that. It looks like there are still some tidy related test failures. If you have https://metacpan.org/pod/distribution/Code-TidyAll/bin/tidyall installed or don't mind installing it, can you run tidyall over your branch? If you would prefer not to install it, let me know and I can just merge and tidy it up afterwards.

**Comment by lharey** [comment]
Hi 

I have now run tidyall on the files and updated

Regards
Lisa

**Comment by klp2** [comment]
Cool, thank you!

**Pull request merged**